verification/curator-service/ui/src/components/ViewCase.tsx

```diff
@@ -134,344 +145,420 @@ function dateRange(range?: { start?: string; end?: string }): string {
 function CaseDetails(props: CaseDetailsProps): JSX.Element {
     const classes = useStyles();
     return (
-        <Container maxWidth="md" className={classes.container}>
-            <Typography className={classes.caseTitle} variant="h5">
-                Case {props.c._id}{' '}
-                {props.enableEdit && (
-                    <Link
-                        to={`/cases/edit/${props.c._id}`}
-                        style={{ textDecoration: 'none' }}
-                    >
-                        <Button
-                            variant="outlined"
-                            color="primary"
-                            className={classes.editBtn}
-                            endIcon={<EditIcon />}
+        <>
+            <nav className={classes.navMenu}>
+                <TypoLink variant="body2" href="#case-data">
+                    CASE DATA
+                </TypoLink>
+                <br />
+                <TypoLink variant="body2" href="#demographics">
+                    DEMOGRAPHICS
+                </TypoLink>
+                <br />
+                <TypoLink variant="body2" href="#location">
+                    LOCATION
+                </TypoLink>
+                <br />
+                <TypoLink variant="body2" href="#event-history">
+                    EVENT HISTORY
+                </TypoLink>
+                <br />
+                <TypoLink variant="body2" href="#symptoms">
+                    SYMPTOMS
+                </TypoLink>
+                <br />
+                <TypoLink variant="body2" href="#travel-history">
+                    TRAVEL HISTORY
+                </TypoLink>
+                <br />
+                <TypoLink variant="body2" href="#pathogens">
+                    PATHOGENS
+                </TypoLink>
```

Comment:
A couple sections are missing. Here are the ones from Edit case:
![Screen Shot 2020-07-27 at 2 00 23 PM](https://user-images.githubusercontent.com/30459667/88544619-94f00200-d011-11ea-8362-f0bfc5fddc1a.png)

Reply:
as discussed, made it consistent with the mocks.

Reply:
preexisting conditions is missing from mocks and is also quite long, I've added transmission to match the mocks .